core/server/data/export/index.js

```diff
@@ -36,7 +36,7 @@ exportFileName = function exportFileName() {
 
 getVersionAndTables = function getVersionAndTables() {
     var props = {
-        version: versioning.getDatabaseVersion(),
+        version: ghostVersion.full,
```

Comment:
In LTS branch we have read the database version from the database when exporting the database.
When working with knex-migrator, it's only interesting how your migrations table looks like and what your ghost version is. From there on knex-migrator decides which migration scripts are missing. 
Just keeping the version info in the meta field.